src/DIRAC/MonitoringSystem/Client/Types/PilotsHistory.py

```diff
@@ -0,0 +1,36 @@
+""" Definition for PilotsHistory Monitoring type.
+    Filled by the agent "WorkloadManagement/StatesAccountingAgent"
+"""
+
+from DIRAC.MonitoringSystem.Client.Types.BaseType import BaseType
+
+
+class PilotsHistory(BaseType):
+    """
+    .. class:: PilotsHistoryMonitorType
+    """
+
+    def __init__(self):
+        """
+        :param self: self reference
+        """
+
+        super().__init__()
+
+        self.keyFields = ["TaskQueueID", "GridSite", "GridType", "Status"]
+
+        self.monitoringFields = ["Pilots"]
```

Comment:
It is the numerical field that we monitor. The other ones, the key fields, are the ones that we use to group the monitoringFields when we plot them.
So when we plot this, it will be something like "Number of Pilots by GridSite", for example

Reply:
Should we call the monitoring fields something else? Like `NumOfPilots` or `NumPilots`?

Reply:
Yes, would be better.